.nuspec

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+    <metadata>
+        <id>Messerli.Utility</id>
+        <version>0.1.1</version>
+        <authors>Jan Nils Ferner, Dominic Groth, Thomas Bruderer</authors>

```

Comment:
Add yourself to author list if you want?

Reply:
a0584de71a3a0e535518b43ef4d0a7b2bec2d577